.github/ISSUE_TEMPLATE/request-package-update.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Package Update request
+description: Request an update to a package available in the repository.
+labels:
+  - Package Request
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Name
+      description: Name of the software.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Requested version.
+    validations:
+      required: true
```

Comment:
When the package is updated, it should be updated to the latest stable version anyway, barring build issues, issues with dependencies or reverse dependencies, etc. Knowing that, is this field really helpful?

Reply:
People might request a specific version for specific reasons, so I'd like to keep the field. Definitely shouldn't be required though.

Reply:
Good enough reason for me.